app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "sample app", "#", id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home", "#" %></li>
+        <li><%= link_to "Help", "#" %></li>
+        <li><%= link_to "Log in", "#" %></li>
```

Comment:
Tìm hiểu và sử dụng i18n nhé, cho toàn bộ text hiển thị ở view của end user 

ngoài ra `"#"` -> `static_pages_home_path` nhỉ

tìm hiểu lệnh `rails routes` để lấy prefix thêm `_path` nhé

Reply:
ready